samples/uikit/README.md

```diff
@@ -5,22 +5,14 @@ Apple devices, such as an iPhone.
 
 To build and run the sample do the following:
 
-1.  Open `samples/uikit/konan.xcodeproj` with Xcode.
+1.  Open  `UIKitSample.xcodeproj` with Xcode and fix bundle ID and development team.
+  or
+1a.  Fix bundleIdPrefix and DEVELOPMENT_TEAM in project.yml and generate XCode project
+    with xcodegen (https://github.com/yonaskolb/XcodeGen/).
 
-2.  Open the project's target through project navigator, go to tab 'General'.
-    In 'Identity' section change the bundle ID to the unique string in
-    reverse-DNS format. Then select the team in 'Signing' section.
+2.  Now build and run the application on a connected iPhone with Xcode or simulator.
 
-    See the
-    [Xcode documentation](https://developer.apple.com/library/content/documentation/IDEs/Conceptual/AppDistributionGuide/ConfiguringYourApp/ConfiguringYourApp.html#//apple_ref/doc/uid/TP40012582-CH28-SW2)
-    for more info.
+Note that in this example we do not use storyboards, and instead create user interface
```

Comment:
So it is not clear anymore whether Kotlin is compatible with storyboards. I guess this should be stated explicitly.

Reply:
Storyboard is data format, not sure what do you mean by language compatibility with data format. Could you please suggest the verbatim statement you'd like to make here.

Reply:
I mean that it is possible to make pure Kotlin application with storyboards used for UI.
It is likely enough to add additional sense like "Defining UI with storyboards in pure Kotlin iOS applications is supported too".